rfcs/20220310-hip-support/README.md

```diff
@@ -0,0 +1,172 @@
+# AMD backend support (RFC)
+
+## Introduction
+The idea for this RFC is to expose the AMD backend for oneDNN.
+- This branch introduces HIP backend support for the primitives that are currently supported for CUDA backend.
+- Testing and Performance analysis are yet to be done.
+- Build process and compilation is successfully validated.
+
+## Proposal
+The primitives are built using MIOpen and hipDNN library, which are the open-source DNN libraries of AMD.
```

Comment:
The `hipDNN` library has been in beta stage for 3 years or so. It doesn't look like there is a plan to mature the project.
Is that possible to always use MIOpen directly?